pom.xml

```diff
@@ -15,7 +15,7 @@
         <changelist>999999-SNAPSHOT</changelist>
         <gitHubRepo>jenkinsci/pipeline-graph-view-plugin</gitHubRepo>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.277.1</jenkins.version>
+        <jenkins.version>2.321</jenkins.version>
```

Comment:
do you need a version this new? is there a reason to not target an LTS? is it because of some css classes?

Reply:
Table view style is not available in current LTS.

Reply:
right, it looks ~fine without the styles so I think we could just either leave this alone or bump to a more recent LTS.

i.e. keep the styles but only people on recent versions will get them

Reply:
Is there a simple way to extract the table styles to a CSS file that plugins can load? I also have the same problem in the warnings plugin. (For the new tooltips I copied the new CSS into one of my CSS files, but for tables the change seems to be bigger)

Reply:
short of copying the styles no but it's likely error prone, and for this case not needed.

as it looks ok without it.

Reply:
```suggestion
        <jenkins.version>2.277.1</jenkins.version>
```

Reply:
Latest LTS is ` 2.319.1` btw